plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/TransportReindexAction.java

```diff
@@ -19,6 +19,8 @@
 
 package org.elasticsearch.plugin.reindex;
 
+import java.util.Objects;
```

Comment:
Oh it just moved, I think this was erroneously moved though, if we follow our import rules